builder/.babelrc

```diff
@@ -1,19 +1,20 @@
 {
   "presets": [
-    ["env", {
-      "modules": false,
-      "forceAllTransforms": true,
-      "targets": {
-        "browsers": "> 1%",
-        "uglify": true
```

Comment:
Is there any pro/con argument about the uglify-part?

Reply:
After updating `@babel/preset-env` dependency, this options was deprecated. You can see it here [@babel/preset-env](https://babeljs.io/docs/en/babel-preset-env#targets).